actionview/app/assets/javascripts/README.md

```diff
@@ -36,6 +36,16 @@ Require `rails-ujs` into your application.js manifest.
 //= require rails-ujs
 ```
 
+Usage with yarn/webpacker
```

Comment:
Sure will add, although thought it was important to mention yarn and webpacker as that's the recommended way now isn't it?.